typescript/common-resolvers/src/export/format-coco/add-image-dimensions-to-labels.ts

```diff
@@ -1,22 +1,28 @@
-import { DbLabel, Repository } from "../../types";
-import { DbLabelWithImageDimensions } from "./coco-core/types";
+import { DbImage } from "../..";
+import { DbLabel } from "../../types";
 
-export const addImageDimensionsToLabels = async (
+type ImageDimensions = {
```

Comment:
If you're OK, let's rename and move the type into a `geometry` directory of @labelflow/utils. We will populate the directory with all geometry-related types and functions and eventually switch easier to a dedicated library if it becomes too big..
No need to make sure it's used everywhere for now but it will let us reuse it anywhere a 2D size is needed 🚀 

```suggestion
export type Size2D = {
```

Reply:
Actually it's a small part of the `DbImage` type so maybe I could just `Pick` from it to reflect that it doesn't come out of nowhere

Reply:
Yes, let's pick the type for now, it's simpler 👍